plugins/chat/db/migrate/20230411032630_add_thread_not_deleted_index_chat_messages.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AddThreadNotDeletedIndexChatMessages < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    execute <<~SQL
```

Comment:
Note that we have to drop the index if it exists when disabling transaction since creating an index concurrently may result in an invalid index.

Reply:
Ah yep sorry I do forget that quite often since its a "newer" rule of ours, will fix